Makefile

```diff
@@ -1,7 +1,7 @@
 NAME    := dpdk
 SRC_EXT := xz
 SOURCE   = http://fast.dpdk.org/rel/$(NAME)-$(VERSION).tar.$(SRC_EXT)
```

Comment:
You can actually remove this line.  It's not needed with the updated `packaging/`.